src/collectd.conf.pod

```diff
@@ -860,6 +860,18 @@ When the AMQP1 connection is lost, defines the time in seconds to wait
 before attempting to reconnect. Defaults to 1, which implies attempt
 to reconnect at 1 second intervals.
 
+=item B<SendQueueLimit> I<SendQueueLimit>
+If there is no AMQP1 connection, the plugin will continue to queue
+messages to send, which could result in unbounded memory consumption. This
+parameter is used to limit the number of messages in the outbound queue to
+the specified value. The default value is 0, which disables this feature.
+
+=item B<SendQueueDropNew> I<SendQueueDropNew>
```

Comment:
Why we need this option and behaviour configurable?

Reply:
I thought somebody may want this behavior, but it's possible there is no real use case for it. Chatted with Matthias a bit, and he couldn't think of one. I can remove it for the time being, and if somebody ever needs it, it's easy to add back.

Reply:
I think we don't need to introduce features just for their presence.
After release that should be supported "forever", as we prefer not to introduce possible regressions.

Reply:
> don't need to introduce features just for their presence.

+1, and if at all would like to introduce some feature to test out (with a good reason), then it could/should be clearly marked as experimental (subject to change/remove)